utils/jupyterlite.py

```
@@ -225,32 +225,44 @@ def load_materials_from_folder(folder_path: Optional[str] = None, verbose: bool
+    from mat3ra.made.tools.build_components import MaterialWithBuildMetadata
+
+    name_lower = name.lower()
+    resulting_material = None
+
+    for filename in sorted(os.listdir(folder_path)):
+        if filename.endswith(".json") and name_lower in os.path.splitext(filename)[0].lower():
+            with open(os.path.join(folder_path, filename), "r") as file:
+                data = json.load(file)
+            try:
+                resulting_material = MaterialWithBuildMetadata.create(data)
+            except Exception:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)